tests/transi/transi_test_dirtrans_adjoint.c

```diff
@@ -242,7 +242,13 @@ struct DirTransAdj_t dirtrans_adj = new_dirtrans_adj(&trans);
   dirtrans_adj.rspvor    = rspvorx;
   dirtrans_adj.rspdiv    = rspdivx;
   dirtrans_adj.rgp       = rgpy;
+#ifdef GPU_VERSION
+TRANS_CHECK_ERROR(trans_dirtrans_adj(&dirtrans_adj), TRANS_NOTIMPL);
+printf("ERROR: trans_dirtrans_adj not yet implemented for GPU version\n");
+exit(0);
+#else
 TRANS_CHECK( trans_dirtrans_adj(&dirtrans_adj) );
+#endif
```

Comment:
We can remove this now right? PR https://github.com/ecmwf-ifs/ectrans/pull/254 implemented this test for GPU.

Reply:
That is true! Forgot this in the rebase.